c/ACPC10A.c

```diff
@@ -1,18 +1,19 @@
 #include <stdio.h>
 
 int main() {
-	int a, b, c, m;
+	int a, b, c, m,b2;
 	scanf("%d %d %d", &a, &b, &c);
 	
 	while(1)
 	{
+		b2=b*b;
 		if(a == 0 && b == 0 && c == 0)
 			break;
 		else
 		{
 		 if( b == ((a + c) /2.0 ))
 		 	printf("AP %d\n", c+(c-b));
-		 else
+		 else if(b2==a*c)
```

Comment:
Not a necessary check since the problem mentions that the series will either be A.P or G.P.